src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -25,8 +28,21 @@
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
 
-final class NoSuperfluousPhpdocTagsFixer extends AbstractFixer
+final class NoSuperfluousPhpdocTagsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /** @var bool */
+    private $shouldFixMixed = true;
```

Comment:
If you call this `$allowMixed` with default `false`, then you could just do `$this->allowMixed = $this->configuration['allow_mixed_superfluous_phpdoc'];` while resolving the configuration. Only thing then to do is to negate the return in `annotationIsSuperfluous` and then you no longer have a sort of double negation, plus consistency, because currently everything is talking about "allow" in the config and test, but not in the actual code.

Reply:
yea.. great point :+1: